kore/src/Kore/IndexedModule/Resolvers.hs

```diff
@@ -89,46 +96,24 @@ getHeadApplicationSorts
     -> SymbolOrAlias level     -- ^the head we want to find sorts for
     -> ApplicationSorts level
 getHeadApplicationSorts m patternHead =
-    case resolveSymbol m headName of
-        Right (_, sentence) ->
-            case symbolOrAliasSorts headParams sentence of
-                Left err     -> error (printError err)
-                Right result -> result
-        Left _ ->
-            case resolveAlias m headName of
-                Right (_, sentence) ->
-                    case symbolOrAliasSorts headParams sentence of
-                        Left err     -> error (printError err)
-                        Right result -> result
-                Left _ ->
-                    error ("Head " ++ show patternHead ++ " not defined.")
+    applyToSentence sentenceSorts m patternHead
```

Comment:
How about `applyToSymbolOrAliasSentence` or `applyToHeadSentence`? People are less familiar with `head` in this context, but it would make it obvious that one can't pass, say, an axiom.

Reply:
Done.